Dockerfile

```diff
@@ -52,4 +52,4 @@ ENV RENKU_SVC_NUM_WORKERS 4
 ENV RENKU_SVC_NUM_THREADS 8
 ENV RENKU_DISABLE_VERSION_CHECK=1
 
-ENTRYPOINT ["renku"]
+ENTRYPOINT ["tini", "-g", "--", "renku"]
```

Comment:
I've used `tini` in the past. Why is it necessary here now?

Reply:
Mostly because of:
> It ensures that the default signal handlers work for the software you run in your Docker image. For example, with Tini, SIGTERM properly terminates your process even if you didn't explicitly install a signal handler for it.

And 

> With the -g option, Tini kills the child process group , so that every process in the group gets the signal. This corresponds more closely to what happens when you do ctrl-C etc. in a terminal: The signal is sent to the foreground process group.

Both from: https://github.com/krallin/tini

I have seen in a few cases in the notebook service specifically that if the process that runs is not with PID 1 then SIGTERM from k8s does not properly reach the python code. And the server is killed forcefully only after the graceful shutdown period of 30 seconds that k8s uses by default. With tini I do not have to worry about this - the server and any child process gets the SIGTERM from k8s. Also with gunicorn/wsgi spawning multiple parallel workers I am not sure they will all get the SIGTERM properly without tini.

Reply:
makes sense, thanks!